src/Hedgehog/Internal/Gen.idr

```diff
@@ -52,6 +52,10 @@ public export
 toTree : Gen a -> Gen (Cotree a)
 toTree (MkGen unGen) = MkGen $ \si,se => pure (unGen si se)
 
+public export
```

Comment:
Please add documentation. Also: Is it necessary to `public export` this?

Reply:
I used `public export` only because all other functions around it upon the `Gen` type, like `toTree` and `mapGen` are also `public export`. Now I see that there is no used of this function, thus I removed it.